src/Core/Services/StateService.cs

```diff
@@ -1359,6 +1359,39 @@ public void SetConfigs(ConfigResponse value)
             _storageMediatorService.Save(Constants.ConfigsKey, value);
         }
 
+        public async Task<Region?> GetActiveUserRegionAsync(string userId = null)
+        {
+            return (await GetAccountAsync(
+                ReconcileOptions(new StorageOptions { UserId = userId }, await GetDefaultStorageOptionsAsync())
+            ))?.Settings?.Region;
+        }
+
+        public async Task UpdateActiveUserEnvironmentAsync(Region region, EnvironmentUrlData urls, string userId = null)
```

Comment:
🎨 Naming issue, if it's the active user then remove the `userId` parameter, otherwise change the name to `UpdateEnvironmentAsync(Region region, EnvironmentUrlData urls, string userId = null)`

Reply:
This was removed and moved into the state migration service.